internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/MockVeniceRouterWrapper.java

```diff
@@ -133,6 +135,7 @@ static StatefulServiceProvider<MockVeniceRouterWrapper> generateService(
           .put(ZOOKEEPER_ADDRESS, zkAddress)
           .put(KAFKA_BOOTSTRAP_SERVERS, kafkaBootstrapServers)
           .put(SSL_TO_STORAGE_NODES, sslToStorageNodes)
+          .put(KAFKA_OVER_SSL, sslToStorageNodes)
```

Comment:
Is this config related to the bug fix?

Reply:
it is related to the problem mentioned in https://github.com/linkedin/venice/pull/1825/files#r2109526430
What is the way for router to know that controller is on secure port or unsecure port? controller has one flag "getSslFactory().isPresent()", but router has two flags sslToKafka and sslToStorageNodes. In this mock, i have to set this flag on.